lametro/models.py

```diff
@@ -454,11 +454,7 @@ def ceo(cls):
     @property
     def headshot_url(self):
         if self.headshot:
-            # Assigning this to image_url would make 'return static(image_url)'
-            # at the end of this property concatenate 'static' to the url.
-            # Returning here solves that.
-            if "headshot_placeholder.png" not in self.headshot.path:
-                return self.headshot
+            return self.headshot.url
```

Comment:
We previously needed to load all images with `static()` because `django-councilmatic` was storing them with static files. Now, since we've told `django-councilmatic` to upload headshots to S3, if there is a headshot, we can return its URL directly, and only get the manual headshots and placeholder from static.